datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -52,7 +53,7 @@ python-binary-memcached==0.26.1; sys_platform != 'win32'
 python-dateutil==2.8.0
 python3-gearman==0.1.0; sys_platform != 'win32' and python_version > '3.0'
 pytz==2019.3
-pyvmomi==v6.5.0.2017.5-1
+pyvmomi==6.7.3
```

Comment:
Yes, that's because of:

> ERROR: vsphere-automation-sdk 1.20.0 has requirement pyVmomi>=6.7, but you'll have pyvmomi 6.5.0.2017.5.post1 which is incompatible.

Reply:
😢